Subject: Bug fix: remove the assertion about the length of the buffer
		      that hasn't been initialized.

Buffer s[BUFSIZE] is declared at the beginning of CHLLCode::appendExp method.
There is assert(strlen(s) < BUFSIZE) at the end of the method.
But in the body of the method the buffer doesn't always get initialized,
therefore at the end the length may be undefined.

>From : Anatoli Koutsevol <Anatoli.Koutsevol@MarkovProcesses.com>

Index: codegen/chllcode.cpp
===================================================================
RCS file: /cvsroot/boomerang/boomerang/codegen/chllcode.cpp,v
retrieving revision 1.52
diff -u -r1.52 chllcode.cpp
--- codegen/chllcode.cpp	21 Jan 2004 06:58:14 -0000	1.52
+++ codegen/chllcode.cpp	1 Feb 2004 20:17:02 -0000
@@ -545,7 +545,6 @@
                 std::endl;
             assert(false);
     }
-    assert(strlen(s) < BUFSIZE);
     assert(strlen(str) < BUFSIZE);
 }
